ios/FluentUI/Card/CardView.swift

```diff
@@ -221,13 +219,27 @@ open class CardView: UIView {
         // Title
         primaryLabel.text = title
         primaryLabel.numberOfLines = twoLineTitle ? Constants.twoLineTitle : Constants.defaultTitleNumberOfLines
-        primaryLabel.textAlignment = .natural
+        switch style {
+        case .horizontal:
+            primaryLabel.textAlignment = .natural
+            primaryLabel.style = .subhead
+        case .vertical:
+            primaryLabel.textAlignment = .center
+            primaryLabel.style = .caption1
+		}
         addSubview(primaryLabel)
 
         // Subtitle
```

Comment:
Do we still need the subtitle? Doesn't look like the new design has one, just the title and icon.

Reply:
Not really but also have no reason to remove it. The changes I made still support having a title, subtitle, or a two line title. 

My changes are being made so the component can be used now but that doesn't mean there isnt another iteration of the changes. I would assume for wider support they might want to keep both title and subtitle, so that's why I left it.